cfgov/scripts/enforcement_transfer.py

```diff
@@ -0,0 +1,191 @@
+        self.reset()
+        self.strict = False
+        self.convert_charrefs = True
+        self.fed = []
+
+    def handle_data(self, data):
+        self.fed.append(data)
+
+    def get_data(self):
+        return ''.join(self.fed)
+
+
+def strip_tags(html):
+    tag_stripper = TagStripper()
+    tag_stripper.feed(html)
+    stripped = tag_stripper.get_data()
+
+    try:
+        data = stripped.decode('utf-8')
+    except (UnicodeEncodeError, AttributeError):
+        data = stripped
+
+    return normalize('NFKC', data).strip()
+
+
+def clean_status(s):
+    return s.replace('-', ' ').replace('udge', 'udg').title()
+
+
+def get_metadata_value(item):
+    return strip_tags(item['value']['blob']) or None
+
+
+def transfer_page(page, parent, errors):
+    url = page.get_url()
+
+    if not url or 'policy-compliance/enforcement/actions' not in url:
+        return
+
+    keys = vars(page)
+
+    old_categories = page.categories.all()
+
+    if not len(old_categories):
+        return errors.append(f'No assigned categories for {url}')
+
+    category = CFGOVPageCategory(name=old_categories[0].name)
+
+    tags = page.tags.all()
+    sidefoot = get_stream_data(page, 'sidefoot')
+    sidefoot_fields = []
+
+    court = ''
+    docket_number = None
+    status = None
+    institution_type = None
+    date_filed = None
+
+    for field in sidefoot:
+        if field['type'] == 'related_metadata':
+            metadata = field['value']['content']
+            for item in metadata:
+                heading = item['value']['heading']
+
+                try:
+                    if heading == 'Court':
+                        court = get_metadata_value(item, heading)
+                    elif heading in ['Institution type', 'Institution']:
+                        institution_type = get_metadata_value(
+                            item,
+                            heading
+                        )
+                    elif heading == 'Status':
+                        listed_status = clean_status(
+                            get_metadata_value(item, heading)
+                        )
+                        if listed_status not in STATUS_OPTIONS:
+                            status = STATUS_OPTIONS[0]
+                        else:
+                            status = listed_status
+                    elif heading.lower() in [
+                        'docket number',
+                        'case number',
+                        'case numbers',
+                        'file number',
+                        'file numbers',
+                        'civil action number'
+                    ]:
+                        docket_number = item['value']['blob']
+                    elif heading == 'Date filed':
+                        date_filed = item['value']['date']
+                    elif (
+                        heading == 'Category' or
+                        heading == 'Topics' or
+                        heading == 'Last updated'
```

Comment:
I'm a bit concerned about losing the data held in the "Last updated" or "Date updated" field. Are you sure we don't need it?

Reply:
These fields are only on a few pages and are universally out of date, that is, there have been revisions since the "Last Updated" date. If this type of information is valuable, seems like it would be better to hook into the latest revision rather maintain a separate arbitrary date.

That said, I have nonetheless logged the few cases in case it needs to be reintroduced.

Reply:
The stakeholder confirmed that we do not need to preserve the "date updated" field on the page, so this looks good.